### PR TITLE
[usbdev/doc] Add PMOD info for testing usbdev on FPGA

### LINKED ISSUE #1653: opentitan usb device interface
Opentitan on Nexys-Video incorporates a USB device that makes use of PMOD connector JB to provide the USB connector, pullup resistor and sense input. Is this a custom item because I don't seem to be able to find it in the Digilent PMOD catalogue ?

### PR DESCRIPTION
**Pull request by mdhayter**
The usbdev testing on the Nexys Video board requires use of a PMOD.
Add description of this card to the main documentation and the
Kicad design files.

Fixes #1653

Signed-off-by: Mark Hayter <mark.hayter@gmail.com>

### CODE REVIEW

**Comment by mdhayter** [comment]
Finally adding the PMOD information. Please check the adding the pmod directory is the correct thing to do here. It seemed to me better than hiding under doc. Alternatively could name it pcb?

Seems to work in a local docs build.

File: hw/ip/usbdev/doc/_index.md [hunk 1]
```diff
@@ -144,6 +144,28 @@ To control the pullup on the D- line, this USB device features `dn_pullup_o` sig
 Of the two pullup signals `dp_pullup_o` and `dn_pullup_o`, only one can be enabled at any time.
 As this is a Full-Speed device, `dp_pullup_o`, i.e., the pullup on D+ is used by default (`pinflip` equals zero).
 
+### FPGA board implementation with PMOD
```
Comment (vogelpi) [inline hw/ip/usbdev/doc/_index.md:147]: Nit: I would capitalize the first letter of every word in this heading (to match the style with other headings in the doc)

File: hw/ip/usbdev/doc/_index.md [hunk 2]
```diff
@@ -144,6 +144,28 @@ To control the pullup on the D- line, this USB device features `dn_pullup_o` sig
...
+
+The interface was developed using the Digilent Nexys Video board with a PMOD card attached.
+A PMOD interface with direct connection to the SoC should be used (some PMOD interfaces include 100R series resistors which break the signal requirements for USB).
+The PMOD card includes two USB micro-B connectors and allows two USB interfaces to be used.
+The D+ and D- signals have 22R series resistors (in line with the USB spec) and there is a 1.5k pullup on D+ to the pullup enable signal.
+There is a resistive divider to set the sense pin at half of the VBUS voltage which enables detection on the FPGA without overvoltage on the pin.
+
+![PMOD Schematic](dualpmod-sch.svg)
+
+The PMOD pcb is [available from OSH Park](https://oshpark.com/shared_projects/xMKhTIHn).
```
Comment (vogelpi) [inline hw/ip/usbdev/doc/_index.md:157]: Nit: pcb -> PCB?

File: hw/ip/usbdev/doc/_index.md [hunk 3]
```diff
@@ -144,6 +144,28 @@ To control the pullup on the D- line, this USB device features `dn_pullup_o` sig
...
+
+The PMOD design files for Kicad version 5 are in the `usbdev/pmod` directory.
```
Comment (vogelpi) [inline hw/ip/usbdev/doc/_index.md:159]: `usbdev/pmod` -> [`usbdev/pmod`](https://github.com/lowRISC/opentitan/tree/master/hw/ip/usbdev/pmod)

With the embedded link (sorry, don't know how to un-render this).

Comment (vogelpi) [reply hw/ip/usbdev/doc/_index.md:159]: Kicad -> KiCad?

File: hw/ip/usbdev/doc/_index.md (hunk 1 above)
Comment (mdhayter) [reply hw/ip/usbdev/doc/_index.md:147]: done

File: hw/ip/usbdev/doc/_index.md (hunk 2 above)
Comment (mdhayter) [reply hw/ip/usbdev/doc/_index.md:157]: done

File: hw/ip/usbdev/doc/_index.md (hunk 3 above)
Comment (mdhayter) [reply hw/ip/usbdev/doc/_index.md:159]: done

**Comment by mdhayter** [comment]
Pushing a new version fixing the nits. For the other questions:

1) I don't think that I know enough about the KiCad files (all generated by the tool) to edit them to add a license. I was assuming coverage from the blanket "Unless otherwise noted, everything in this repository is covered by the Apache License, Version 2.0 (see LICENSE for full text)." in the readme at the top of the repo. I have added a README that includes the license info.

2) I did a diff and couldn't see any reason to keep the backup files, so will delete.

**Pull request merged**